INSTALLATION.md

```diff
@@ -143,7 +144,7 @@ For the account you created:-
 
         userType: "SUPERADMIN"
 
-Refer to [Talawa-api-mongodb-section](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md#instructions-to-edit-records-for-admin-user) for instructions on the same.
+Refer to INSTALLATION.md section of `instructions-to-edit-records-for-admin-user` of [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api) for instructions on the same.
```

Comment:
1. Remove this `Elevate Permissions` section. If the person has followed the API setup it would already have been done. 
1. The issue is that if it is in only one place then it will always be up to date. When this is in two places errors will be introduced.

Reply:
Okay.